patches/tools-perf-page_sets-system_health-system_health_stories.py.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tools/perf/page_sets/system_health/system_health_stories.py b/tools/perf/page_sets/system_health/system_health_stories.py
+index 8889e230b5e8fb53f81b09b07a30f83da0b22067..03352444d8425329b6b52de766de79ff7af36018 100644
+--- a/tools/perf/page_sets/system_health/system_health_stories.py
++++ b/tools/perf/page_sets/system_health/system_health_stories.py
+@@ -26,7 +26,7 @@ class SystemHealthStorySet(story.StorySet):
+                take_memory_measurement=False,
+                tag=None):
+     super(SystemHealthStorySet, self).__init__(
+-        archive_data_file=('../data/system_health_%s.json' % platform),
++        archive_data_file=('../../../../brave/tools/perf/page_sets/data/system_health_%s.json' % platform),
```

Comment:
can't we just use the same path here? Or symlink in the config scripts?

Reply:
Didn't get your point.
The original is a file committed in `src/`. Our file has a bunch of changes. It's 100+ lines patch to change the original one.
 
Deleting and making a symlink will be also conflict and look obscure.

Reply:
sorry, initially read this as an output file, not input

Reply:
actually re https://github.com/brave/brave-core/pull/14693/files#r980331402 - wouldn't this make running the chromium tests impossible or can it use our version of the file?

Reply:
The archives in `system_health_%s.json` is a private things (chrome-partner bucket).
So it's already impossible to run these tests without a special access from Google.
We have to replace only the archives (WPRs) to own recorded.